extensions/sensors/SensorBase.h

```diff
@@ -64,17 +61,17 @@ class SensorBase : public core::Processor {
   void onSchedule(const std::shared_ptr<core::ProcessContext> &context, const std::shared_ptr<core::ProcessSessionFactory> &sessionFactory) override;
 
   class WriteCallback : public OutputStreamCallback {
-     public:
-      explicit WriteCallback(std::string data)
-          : data_{std::move(data)}
-      {}
-      std::string data_;
-      int64_t process(const std::shared_ptr<io::BaseStream>& stream) override {
-        if (data_.empty()) return 0;
-        const auto write_ret = stream->write(reinterpret_cast<const uint8_t*>(data_.data()), data_.size());
-        return io::isError(write_ret) ? -1 : gsl::narrow<int64_t>(write_ret);
-      }
-    };
+   public:
+    explicit WriteCallback(std::string data)
+        : data_{std::move(data)} {
+    }
+    std::string data_;
+    int64_t process(const std::shared_ptr<io::BaseStream>& stream) override {
+      if (data_.empty()) return 0;
+      const auto write_ret = stream->write(reinterpret_cast<const uint8_t*>(data_.data()), data_.size());
+      return io::isError(write_ret) ? -1 : gsl::narrow<int64_t>(write_ret);
+    }
+};
```

Comment:
this should be indented; I would also add a blank line after it
```suggestion
  };

```